### ISSUE TITLE
Installation issues with latest version of React

### ISSUE THREAD

**Issue by Carpetfizz** [Flipboard/react-canvas#116]
Hello,

I tried installing `react-canvas` today and received the following error during installation:

```
npm WARN peerDependencies The peer dependency react@^0.13.0-beta.1 included from react-canvas will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Linux 3.19.0-32-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "react-canvas"
npm ERR! node v4.2.2
npm ERR! npm  v2.14.7
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react@0.14.2 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-canvas@0.0.1 wants react@^0.13.0-beta.1
npm ERR! peerinvalid Peer react-dom@0.14.2 wants react@^0.14.2

npm ERR! Please include the following file with any support request:
```

After requiring `react-canvas` in my application, I got the following error in the Chrome developer console.

`Error: Cannot find module 'react/lib/invariant' from '/path/to/app/node_modules/react-canvas/lib'`

I would appreciate any help in resolving this issue - thanks!

**Comment by migreva** [comment]
I've also had a similar issue, I downgraded to a 0.13.3 and it worked fine.

[react@0.14](http://facebook.github.io/react/blog/2015/09/10/react-v0.14-rc1.html) was a fairly major release, anyone have any idea when react-canvas will be upgraded to support 0.14?

**Comment by Carpetfizz** [comment]
@migreva I don't mind downgrading to get this to work, but do you think upgrading react later would cause any significant problems?

**Comment by migreva** [comment]
@Carpetfizz i tried upgrading to react 0.14 and there was an error, react-canvas was referencing something in react that no longer existed, it seemed. Don't exactly remember what it was, but i figured that wouldn't be fixed until react-canvas updates to support 0.14

**Comment by gre** [comment]
should be fixed by 1.0.0
